Sources/AVPlayer+Android.swift

```diff
@@ -6,7 +6,9 @@
 //  Copyright © 2017 flowkey. All rights reserved.
 //
 
+#if os(Android)
```

Comment:
I think this entire file should be Android only, no? It would not work anywhere else